From: Christian Kastner <debian@kvr.at>
Date: Mon, 26 May 2014 09:54:57 +0200
Subject: [PATCH] Remove RHEL-specific feature test

The release number comparison breaks  with sid's release number of "unstable".

Forwarded: no
Last-Update: 2015-05-26

---
 tests/keyctl/session/valid/runtest.sh | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/tests/keyctl/session/valid/runtest.sh b/tests/keyctl/session/valid/runtest.sh
index 172343a..b8372ca 100644
--- a/tests/keyctl/session/valid/runtest.sh
+++ b/tests/keyctl/session/valid/runtest.sh
@@ -9,21 +9,6 @@
 result=PASS
 echo "++++ BEGINNING TEST" >$OUTPUTFILE
 
-# describe the keyring created for an anonymous session
-if [ $OSDIST = RHEL -a $OSRELEASE -le 5 ]
-then
-    marker "ANON SESSION"
-    new_session - keyctl rdescribe @s "@"
-    expect_key_rdesc rdesc "keyring@.*@.*@.*@_ses[^@]*\$"
-
-    # check the session keyring ID is shown
-    seskeyring="`tail -2 $OUTPUTFILE | head -1`"
-    if ! expr "$seskeyring" : "Joined session keyring: [0-9]*" >&/dev/null
-    then
-	failed
-    fi
-fi
-
 # describe the keyring created for a named session
 marker "NAMED SESSION"
 new_session qwerty keyctl rdescribe @s "@"
-- 
2.0.0.rc4

